x/pool-incentives/keeper/distr_test.go

```diff
@@ -56,280 +56,299 @@ func (suite *KeeperTestSuite) TestAllocateAssetToCommunityPoolWhenNoDistrRecords
 }
 
 func (suite *KeeperTestSuite) TestAllocateAsset() {
-	keeper := suite.App.PoolIncentivesKeeper
-	mintKeeper := suite.App.MintKeeper
-	params := suite.App.MintKeeper.GetParams(suite.Ctx)
-	params.WeightedDeveloperRewardsReceivers = []minttypes.WeightedAddress{
+	tests := []struct {
+		name                   string
+		args                   []types.DistrRecord
+		mintedCoins            sdk.Coin
+		expectedGaugesBalances []sdk.Coins
+		expectedFeeCollector   sdk.Coin
+		expectedCommunityPool  sdk.DecCoin
+	}{
 		{
-			Address: sdk.AccAddress([]byte("addr1---------------")).String(),
-			Weight:  sdk.NewDec(1),
+			name: "Allocated to the gauges proportionally",
+			args: []types.DistrRecord{
+				{
+					GaugeId: 1,
+					Weight:  sdk.NewInt(100),
+				},
+				{
+					GaugeId: 2,
+					Weight:  sdk.NewInt(200),
+				},
+				{
+					GaugeId: 3,
+					Weight:  sdk.NewInt(300),
+				},
+			},
+			mintedCoins: sdk.NewCoin("stake", sdk.NewInt(50000)),
+			expectedGaugesBalances: []sdk.Coins{
+				sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(2500))),
+				sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(4999))),
+				sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(7500))),
+			},
+			expectedFeeCollector:  sdk.NewCoin("stake", sdk.NewInt(20000)),
+			expectedCommunityPool: sdk.NewDecCoin("stake", sdk.NewInt(5000)),
```

Comment:
The math here wasn't obvious to me until I spun up a local testnet and saw that the community pool param is set to 10%, the staking param is set to 40% (can we change this to say staking instead of fee?), and the gauge param (pool incentives) was set to 30%, with the remaining 20 percent going to dev rewards. Can we make this clearer either through comments or by making this math explicit in the test itself?